libstuff/SData.cpp

```diff
@@ -29,54 +26,38 @@ const string& SData::operator[](const string& name) const {
     }
 }
 
-// --------------------------------------------------------------------------
 void SData::clear() {
-    // Erase everything
     methodLine.clear();
     nameValueMap.clear();
     content.clear();
 }
 
-// --------------------------------------------------------------------------
 void SData::erase(const string& name) {
-    // Erase one value
     nameValueMap.erase(name);
 }
 
-// --------------------------------------------------------------------------
-void SData::merge(const STable& rhs) {
-    // Combine the name/value pairs from two datas
-    nameValueMap.insert(rhs.begin(), rhs.end());
+void SData::merge(const STable& table) {
+    nameValueMap.insert(table.begin(), table.end());
 }
 
-// --------------------------------------------------------------------------
 void SData::merge(const SData& rhs) {
-    // Combine two SData into one
     // **FIXME: What do we do with the content?  Where do we use this?
     merge(rhs.nameValueMap);
 }
 
-// --------------------------------------------------------------------------
 bool SData::empty() const {
-    // Returns whether or not this data is empty
     return (methodLine.empty() && nameValueMap.empty() && content.empty());
 }
 
-// --------------------------------------------------------------------------
 bool SData::isSet(const string& name) const {
-    // Returns whether or not a particular value has been set
     return SContains(nameValueMap, name);
 }
 
-// --------------------------------------------------------------------------
-int SData::calc(const string& name) const {
-    // Forcing 32 bitness here.
+int32_t SData::calc(const string& name) const {
```

Comment:
I know I suggested this, but I don't think it's the right solution. I looked, and almost everywhere we use this in bedrock we do:

```
int someVar = someSData.calc("thing");
```

In these cases, calc  should return `int` to match the type being assigned to. Usually this is 32-bits anyway, but if someone happens to build this on a machine where `int` ins 16 or 64 bits, `calc` should still do the right thing and return an `int` here, so the return type should be `int` and the comment shouldn't specify 32-bits.

As an aside, it looks like pretty much everywhere we use `calc` in auth, it's wrong and we should be using `calc64`.